test/Exporter/Test/Integration/Writer/KnpSnappyWriterTest.php

```diff
@@ -0,0 +1,145 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Exporter\Test\Integration\Writer;
+
+use Exporter\Writer\KnpSnappyWriter;
+use Knp\Snappy\Pdf;
+
+/**
+ * Tests the KnpSnapyWriter.
+ *
+ * @author Robin Chalas <robin.chalas@gmail.com>
+ */
+final class KnpSnappyWriterTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @var string relative path to the test file
+     */
+    private $filename;
+
+    /**
+     * @var string absolute path to the wkhtmltopdf binary
+     */
+    private $wkhtmltopdf;
+
+    public function setUp()
+    {
+        $this->pdfGenerator = new Pdf();
+        $this->filename = 'foobar.pdf';
+        $this->wkhtmltopdf = __DIR__.'/../../../../../vendor/h4cc/wkhtmltopdf-amd64/bin/wkhtmltopdf-amd64';
+
+        if (!is_executable($this->wkhtmltopdf)) {
+            $this->markTestSkipped('The wkhtmltopdf binary is not available');
```

Comment:
Should not be a failed? AFAIK, the binary is downloaded from composer so it should be always available for tests.